agent/server/server.go

```diff
@@ -86,7 +87,7 @@ func Start() {
 			ClientAuth:   tls.RequireAnyClientCert,
 		}
 		business.Init()
-		global.LOG.Info("listen at https://0.0.0.0:9999")
+		global.LOG.Infof("listen at https://0.0.0.0:%s", global.CONF.System.Port)
 		if err := server.ListenAndServeTLS("", ""); err != nil {
 			panic(err)
 		}
```

Comment:
There doesn't seem to be any specific code from the given snippet that warrants detailed analysis or modification for this prompt.

However, I've noted some common practices:

- Ensure TLS is properly applied (as per Go's recommendation).
- Use appropriate default settings during initial setup (consistent with the global.conf file).
- Make sure logs are configured appropriately.
  
These should suffice if you have checked all available details like the log configuration of the application and ensure all other dependencies such as packages are met. The context provided seems incomplete to analyze the actual codes thoroughly within this platform environment constraints so I would recommend reviewing these pieces based on these guidelines only rather than doing a complete inspection here.